OpenDreamRuntime/Objects/MetaObjects/DreamMetaObjectClient.cs

```diff
@@ -98,6 +100,12 @@ sealed class DreamMetaObjectClient : IDreamMetaObject {
                 case "key":
                 case "ckey":
                     return new(_dreamManager.GetSessionFromClient(dreamObject).Name);
+                case "computer_id": // FIXME: This is not secure! Whenever RT implements a more robust (heh) method of uniquely identifying computers, replace this impl with that.
+                    MD5 md5 = MD5.Create();
+                    byte[] brown = Encoding.UTF8.GetBytes(_dreamManager.GetSessionFromClient(dreamObject).Name);
```

Comment:
Why not `UserId`?

Reply:
_"The best way to get the right answer on the internet is not to ask a question; it's to post the wrong answer."_

Reply:
not sure if the FIXME still applies

Reply:
The actual origin of HWId is somewhere deep in the dependencies of RT. I don't happen to know if it is (or even supposed to be) difficult to spoof in any way. In the future, we will still want to append other information onto this value to provide the best identification we reasonably can. Servers rely on this stuff to make their ban system work.

Reply:
Wouldn't we append that information on the RT side though? 

Also, SS14 currently uses this for their bans.

Reply:
It looks like HWId is [generated using random numbers upon the first run](https://github.com/space-wizards/RobustToolbox/blob/a700750d9e48045fe4bebf31a35318d19ed09329/Robust.Shared/Network/HWId.cs#L20), and is then cached in the registry. It appears to not work for users not on Windows, so this may not be the best placeholder.

Reply:
Hence why I said to just use a ckey hash for now

Reply:
Aaagh!!